### PR TITLE
OAuth 2.0 Github Added 

### PR DESCRIPTION
**Pull request by shivamarora96**
#131

### CODE REVIEW

File: stackle_api/app/routes.js [hunk 1]
```diff
@@ -14,10 +14,9 @@ const Reply = postModels.Reply;
 
 module.exports = function (app, db) {
 
-	//api
-	//TODO later
```
Comment (tharindupr) [inline stackle_api/app/routes.js]: ??

Comment (shivamarora96) [reply stackle_api/app/routes.js]: @tharindupr  On requesting localhost:8080/login , user will be redirected to credentials page where enduser will enter his GitHub credentials and authorize the app. On success he will be issued a token. Protected resources can only be access using this token.

**Comment by psnmissaka** [comment]
PR seems good. Fix the merge conflict and the formatting issue I have mentioned. Then we can go ahead with the merge.

**Comment by shivamarora96** [comment]
@psnmissaka since I coded more in ES5 that's why I have a habit of using 'var'.

**Comment by psnmissaka** [comment]
@shivamarora96 I think its better if we can use ES6. For future tasks lets use ES6 and let's do some refactoring later for the code written in ES5.

**Comment by phoenixking25** [comment]
@psnmissaka i can help in es6 format!

**Pull request merged**